python/tk_multi_publish2/util.py

```diff
@@ -217,7 +217,7 @@ def get_frame_sequences(folder, extensions=None, frame_spec=None):
     )
 
 
-def get_publish_name(path, sequence=False):
+def get_publish_name(path, sequence=False, version=False, remove_padding=False):
```

Comment:
Do you think we should make removing the version an optional thing? Is there a good use case for publishing with a version number included? 

I don't think `remove_padding` was included in See my comments on: https://github.com/shotgunsoftware/tk-multi-publish2/pull/68. I think we're on the same page here now though.